src/lib/steps.ts

```diff
@@ -183,6 +183,22 @@ export type ReplaceStep = {
   to_replace: any[][];
 };
 
+export type RollupStep = {
+  name: 'rollup';
+  /** the list of hierarchical columns from lowest to highest level */
+  hierarchy: string[];
+  /** the list of columnns to aggregate, with related aggregation function to use */
+  aggregations: AggFunctionStep[];
+  /** Groupby columns if rollup has to bbe performed by groups */
+  groupby?: string[];
+  /** To give a custom name to the output label column */
+  labelCol?: string;
+  /** To give a custom name to the output level column */
+  levelCol?: string;
+  /** To give a custom name to the output parent column */
+  parentCol?: string;
```

Comment:
Should we name that `parentLabelCol`? Reading the description, I was unsure which column of the parent will be used to fill this one.